caproto/_circuit.py

```diff
@@ -104,7 +107,10 @@ def priority(self, priority):
         logger_name = (f"caproto.circ."
                        f"{self.address[0]}:{self.address[1]}."
                        f"{priority}")
-        self.log = logging.getLogger(logger_name)
+        self.raw_log = logging.getLogger(logger_name)
```

Comment:
Can you think of a situation where a user would want to use this instead of using `self.log`? If not, I think it should be a local variable (I mean `raw_log` instead of `self.raw_log`) for simplicity, to keep the public interface as thin as possible.

Reply:
yes

Reply:
Here we need access to private member self.raw_log
https://github.com/NSLS-II/caproto/pull/427/files/167213113d8c44281133ae373a5ef100cd836071#diff-b300d9c5b4c62704501d4130bf571bfeR164